### ISSUE TITLE
Do not display a point when its value is undefined in radar charts

### ISSUE THREAD

**Issue by lovervoorde** [apache/echarts#14886]
### What problem does this feature solve?
NaN, undefined, null etc. are displayed as zero in radar charts, making it impossible to tell whether the value is actually zero or is missing.

I would like it if the point was skipped, and the two points on either side of it were joined with a straight line.

### What does the proposed API look like?
series-radar.connectNulls: true
series-radar.nullsToZero: false

**Comment by lovervoorde** [comment]
Here is a demo of how it could look.
It is made using a line graph on polar axes, which might be a reasonable interim solution for anyone looking for this feature:
https://codepen.io/lovervoorde/pen/zYZvQmv

**Comment by nsclong** [comment]
Yes please folks. I also have the same issue of nulls being shown as zeros on Radar charts, and it would be great to have a fix.
Thanks for the great library.

**Comment by Tangjj1996** [comment]
I have met the same problem.
